docker-compose.yml

```diff
@@ -1,4 +1,15 @@
 services:
+  app:
+    tty: true
+    stdin_open: true
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: development
+    volumes:
+      - .:/home/node/app
+    ports:
+      - "5173:5173"
```

Comment:
```suggestion
```
I suppose you have used it during development, but we usually just run the app in our machine (i.e. in the host itself, without any use of a container). So we don't need it